vscode-dotnet-runtime-extension/src/test/functional/DotnetCoreAcquisitionExtension.test.ts

```diff
@@ -185,6 +201,45 @@ suite('DotnetCoreAcquisitionExtension End to End', function()
     assert.isFalse(fs.existsSync(result!.dotnetPath), 'the dotnet path result does not exist after uninstalling from all owners');
   }
 
+  function includesPathWithLikelyDotnet(pathToCheck : string) : boolean
+  {
+    const lowerPath = pathToCheck.toLowerCase();
+    return lowerPath.includes('dotnet') || lowerPath.includes('program') || lowerPath.includes('share') || lowerPath.includes('bin') || lowerPath.includes('snap') || lowerPath.includes('homebrew');

```

Comment:
So this would consider something like `C:\ProgramData` as potentially pointing to a valid location that contains the `dotnet` host?

Reply:
Yes -- this is for the test and is meant to remove any possible dotnets from the path. It is not production code. This is to prevent dotnet installs on the CI machines from interfering with the results of the tests.